src/Dev/CSVParser.php

```diff
@@ -115,7 +116,7 @@ class CSVParser implements Iterator
      */
     public function __construct($filename, $delimiter = ",", $enclosure = '"')
     {
-        Deprecation::notice('5.0', __CLASS__ . ' is deprecated, use ' . Reader::class . ' instead');
+        Deprecation::notice('4.13.0', 'Use ' . Reader::class . ' instead', Deprecation::SCOPE_CLASS);
```

Comment:
I'm explicitly _not_ wrapping this one in `Deprecation::withNoReplacement()` because there are only a few instances of it in use in our code, and we can wrap _those_ instead. This seems likely to be used pretty frequently in projects so it's good for their uses to pop up in the main deprecation notices.